src/match_taxa_gbif_backbone.Rmd

```diff
@@ -0,0 +1,471 @@
+---
+title: "Match taxa with GBIF Taxonomy Backbone"
+author:
+  - Damiano Oldoni
+  - Lien Reyserhove
+  - Peter Desmet
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    toc: true
+    toc_depth: 3
+    toc_float: true
+    number_sections: true
+    theme: yeti
+    df_print: paged
+knit: (function(input_file, encoding) { rmarkdown::render(input_file, encoding = encoding, output_file = paste0("../docs/",sub(".Rmd", ".html", basename(input_file))))})
+---
+
+# Setup
+
+## Load libraries
+
+```{r load_libs}
+library(odbc)       # To work with database
+library(tidyverse)  # To do data science
+library(here)       # To work with paths
+library(inborutils) # To match taxa (rgbif called in background)
+```
+
+## Load access informations
+
+Retrieve access informations from configuration file:
+
+```{r get_access_infos}
+ibge_bim <- config::get("ibge_bim")
+```
+
+# Read taxa
+
+Read taxa from `./data/interim/taxa.tsv`:
+
+```{r read_taxa_from_tsv_file}
+taxa <- read_tsv(here("data", "interim", "taxa.tsv"),
+                 col_types = cols(
+                   .default = col_character(),
+                   id = col_double(),
+                   parentid = col_double(),
+                   media = col_logical(),
+                   speciesbeparentid = col_double(),
+                   bruenvi_created = col_date(format = "%Y-%m-%d"),
+                   bruenvi_created = col_date(format = "%Y-%m-%d"),
+                   kingdom_id = col_double()
+                   )
+)
+```
+
+# Match taxa to GBIF Taxonomy Backbone
+
+## Match by `acceptedname` + `scientificnameauthorship`
```

Comment:
This should be the same level as `Match by acceptedname only`, `Match without kingdom` and `Match by acceptedname only`

Reply:
Yes, indeed. Thanks.
Done here: https://github.com/inbo/ibge-bim-species/pull/10/commits/b3afdcc755931132a96839c8e3ee6236fcc21000